validator/accounts/v2/accounts_deposit_test.go

```diff
@@ -0,0 +1,220 @@
+package v2
+
+import (
+	"encoding/hex"
+	"errors"
+	"flag"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	v2keymanager "github.com/prysmaticlabs/prysm/validator/keymanager/v2"
+	"github.com/prysmaticlabs/prysm/validator/keymanager/v2/derived"
+	"github.com/urfave/cli/v2"
+)
+
+type depositTestWalletConfig struct {
+	walletDir                   string
+	walletPasswordFile          string
+	eth1KeystoreFile            string
+	eth1KeystorePasswordFile    string
+	eth1PrivateKeyFile          string
+	httpWeb3ProviderFlag        string
+	publicKeysFlag              string
+	depositAllAccountsFlag      bool
+	skipDepositConfirmationFlag bool
+	keymanagerKind              v2keymanager.Kind
+}
+
+func setupWalletCtxforDeposits(
+	tb testing.TB,
```

Comment:
```suggestion
	t testing.T,
```

Don't think we need `b` here. (it's used for benchmark)

Reply:
Will address in a commit soon.  Thanks!